backend/src/BackendOnlyStdLib/LibHttpClient.fs

```diff
@@ -306,11 +306,11 @@ let fns : List<BuiltInFn> =
             }
 
           | Error reqHeadersErr, _ ->
-            uply { return DError(SourceNone, reqHeadersErr) }
+            uply { return DResult(Error(DStr reqHeadersErr)) }
```

Comment:
This is a type-error (which can evenually be caught statically), so this should remain a DError